src/numeric-content.scss

```diff
@@ -0,0 +1,247 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
```

Comment:
Could you check and remove unused imports ?

Reply:
I will! Thanks for pointing out!